sdkv2-ble/src/main/kotlin/com/nohrd/bike/sdkv2/ble/CustomNohrdService.kt

```diff
@@ -0,0 +1,14 @@
+package com.nohrd.bike.sdkv2.ble
+
+import java.util.UUID
+
+/**
+ * Describes the Custom Service used by the Nohrd Bike v2
+ */
+internal object CustomNohrdService {
```

Comment:
These are unused?

Reply:
Or should probably not be `internal` 🤔

Reply:
yes you're right. I added these last minute